PWGHF/TableProducer/HFLcK0sPCandidateSelector.cxx

```diff
@@ -25,22 +26,8 @@
 using namespace o2;
 using namespace o2::framework;
 using namespace o2::aod::hf_cand_casc;
+using namespace o2::analysis::hf_cuts_lc_tok0sp;
 
-static const int nPtBins = 8;
-static const int nCutVars = 8;
-//temporary until 2D array in configurable is solved - then move to json
-//mK0s(GeV)     mLambdas(GeV)    mGammas(GeV)    ptp     ptK0sdau     pTLc     d0p     d0K0
-constexpr double cuts[nPtBins][nCutVars] = {{0.008, 0.005, 0.1, 0.5, 0.3, 0.6, 0.05, 999999.},  // 1 < pt < 2
-                                            {0.008, 0.005, 0.1, 0.5, 0.4, 1.3, 0.05, 999999.},  // 2 < pt < 3
-                                            {0.009, 0.005, 0.1, 0.6, 0.4, 1.3, 0.05, 999999.},  // 3 < pt < 4
-                                            {0.011, 0.005, 0.1, 0.6, 0.4, 1.4, 0.05, 999999.},  // 4 < pt < 5
-                                            {0.013, 0.005, 0.1, 0.6, 0.4, 1.4, 0.06, 999999.},  // 5 < pt < 6
-                                            {0.013, 0.005, 0.1, 0.9, 0.4, 1.6, 0.09, 999999.},  // 6 < pt < 8
-                                            {0.016, 0.005, 0.1, 0.9, 0.4, 1.7, 0.10, 999999.},  // 8 < pt < 12
-                                            {0.019, 0.005, 0.1, 1.0, 0.4, 1.9, 0.20, 999999.}}; // 12 < pt < 24
-/// Struct for applying D0 selection cuts
-
-//#define MY_DEBUG
```

Comment:
This switches on the debugging pieces of code if uncommented.
If you prefer to remove all the `MY_DEBUG`-related stuff though, I'm fine with it.

Reply:
included the line //#define MY_DEBUG again and left all the debug stuff for the moment. might still be useful at some point